extensions/amp-audioburst-audio/0.1/amp-audioburst-audio.js

```diff
@@ -0,0 +1,460 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Embeds an Audioburst audio player
+ * @author Audioburst
+ *
+ * Example:
+ * <code>
+ * <amp-audioburst-audio
+ *  autoplay
+ *  layout="fixed-height"
+ *  height="315"
+ *  src="https://sapi.audioburst.com/audio/repo/play/web/l1AXp9wBR57y.mp3"
+ *  fullShow="http://storageaudiobursts.blob.core.windows.net/temp/11760_2018081122_t.mp3"
+ *  fullShowPosition="640">
+ * </amp-audioburst-audio>
+ */
+
+import {Layout} from '../../../src/layout';
+import {Services} from '../../../src/services';
+import {listen} from '../../../src/event-helper';
+import {setStyle, setStyles} from '../../../src/style';
+
+
+export class AmpAudioburstAudio extends AMP.BaseElement {
+
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private */
+    this.viewport_ = null;
+    /** @private {string} */
+    this.burstSrc_ = undefined;
+    /** @private {string} */
+    this.fullShowSrc_ = undefined;
+    /** @private {number} */
+    this.fullShowPosition_ = 0;
+    /** @public {boolean} */
```

Comment:
Is the `isPlaying` field referenced by other classes outside this file? If not, it and the others should be `@private` instead. If it needs to be visible for testing, you can use `@visibleForTesting` in addition to `@private`

Reply:
done